spec/server/functional/prepaid.spec.coffee

```diff
@@ -84,6 +84,146 @@ describe 'GET /db/prepaid', ->
       expect(body.length).toEqual(4)
       done()
 
+describe 'GET /db/prepaid/:handle/creator', ->
+  beforeEach utils.wrap (done) ->
+    yield utils.clearModels([Course, CourseInstance, Payment, Prepaid, User])
+    @creator = yield utils.initUser({role: 'teacher'})
+    @joiner = yield utils.initUser({role: 'teacher'})
+    @admin = yield utils.initAdmin()
+    yield utils.loginUser(@admin)
+    @prepaid = yield utils.makePrepaid({ creator: @creator.id })
+    yield utils.loginUser(@creator)
+    yield utils.addJoinerToPrepaid(@prepaid, @joiner)
+    @url = getURL("/db/prepaid/#{@prepaid.id}/creator")
+    done()
```

Comment:
`utils.wrap` is smart enough now that you don't have to include `done` if all async work is being yielded (it calls `done` when the test promise resolves).